packages/material-ui/src/Menu/Menu.js

```diff
@@ -38,11 +38,11 @@ class Menu extends React.Component {
   }
 
   getContentAnchorEl = () => {
-    if (!this.menuListRef || !this.menuListRef.selectedItemRef) {
-      return ReactDOM.findDOMNode(this.menuListRef).firstChild;
+    if (this.menuListRef.selectedItemRef) {
```

Comment:
This should've been more defensive.

Reply:
Why is `this.menuListRef` not present?

Reply:
The formal definition for any ref is `SomeInstance | null`. It is a good question why it is null in that particular stack trace but I would always prefer to a) Make the compiler happy b) follow the official docs and c) throw with something more helpful than a nullpointer exception.

I'm not sure about that particular instance but currently Popover is calling this in it's render method and I'm not sure if MenuList is mounted at that point. So we could dig into all the internals affected or trust the type definition and the compiler.

Reply:
By beeing defensive, we accept that this issue will never be fixed. That the select focus might not be correct.